Gopkg.lock

```diff
@@ -112,6 +112,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "454aacd6dc7c6819166ecc75a75483f4bcebc23ab61c4b2b32e694ee67020d52"
+  inputs-digest = "33d8d0c2675c0851febc914bfcadef2a117d2cc81db1cfea6ca00da527543f18"
```

Comment:
why did this change?

Reply:
oh thats becoz I had introduced some dependencies in an earlier commit but removed them now

Reply:
still need to undo the deps change